docs/development-process/standard-release-process.md

```diff
@@ -0,0 +1,59 @@
+# Standard Release Process
+
+The following document is a description of the standard release process of
+Zarhus-related images. Precise steps and differences from the standard
+process will be described in layer-specific documentation.
+
+## Process steps
+
+1. Make sure that everything that should go into the given release is merged
+   into `main`.
+2. Bump `DISTRO_VERSION` in the `conf/distro/<distro-name>.conf` file.
+3. Fill up the `CHANGELOG.md` file with latest changes.
+    - Run `generate-changelog.sh` to generate the change notes for your release.
+    - Use the `generate-changed-recipes` script to print the list of changes in
```

Comment:
Where are those 2 scripts? I don't see them in `meta-zarhus`

Reply:
You're right. I added them https://github.com/zarhus/meta-zarhus/pull/41/